ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Get_Vendor_Files/tasks/main.yml

```diff
@@ -0,0 +1,19 @@
+---
+##############
+# VENDOR_FILES
+##############
+
+- name: check out AdoptOpenJDK/secrets
+  local_action: git repo=git@github.com:AdoptOpenJDK/secrets.git dest=vendor_files
+
+- name: check if dotgpg is installed
+  local_action: command command -v dotgpg
```

Comment:
Can we change these to `shell command -v dotgpg`. For whatever reason, the `command` builtin module couldn't find `dotgpg` installed on my laptop, but the `shell` module could.

Reply:
done